macros/edr/alerts/anomaly_detection_description.sql

```diff
@@ -19,4 +20,10 @@
 {% macro column_metric_description() %}
     'In column ' || column_name || ', the last ' || metric_name || ' value is ' || round(metric_value,3) ||
     '. The average for this metric is ' || round(training_avg,3) || '.'
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
+
+{% macro dimension_metric_description() %}
+    'The last ' || metric_name || ' value for dimension ' || dimension || ' - ' ||
+    case when dimension_value is null then 'NULL' else dimension_value end || ' is ' || round(metric_value,3) ||
```

Comment:
Why are you checking null here as well? The macro is not invoked if the dimension is null

Reply:
If a column of the dimension allows null as a value it could be null.
In that case the concat will fail and return null, so we need to check it :)